http.go

```diff
@@ -346,93 +426,20 @@ func httpPutViolations(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(http.StatusBadRequest)
 		return
 	}
-	httpPutViolationsInner(w, r, typestr, vs)
-}
-
-func httpPutViolationsInner(w http.ResponseWriter, r *http.Request, typestr string, vs []ViolationRequest) {
+	failedVs := []ViolationRequest{}
```

Comment:
I'm unsure about the use of `ViolationRequest`. It would be nice to include the error message of each failure in the response.